### ISSUE TITLE
missed credentials.py

### ISSUE THREAD

**Issue by timwukp** [datacenter/acitoolkit#39]
hi team,

The latest version looks miss credentials.py. 

Rgds,
tim

**Comment by mtimm** [comment]
credentials.py was purposely removed via bb6d95d9c604816cc38fe637324f67f1a7ad6c36

**Comment by michsmit99** [comment]
Yes, we purposely removed it.  However, you can continue to use a credentials.py file.  

We cleaned up the acceptance of credentials to also include prompts from the user if not provided.  Previously, if you didn't edit credentials.py, it would use the defaults in the file coming from the repository which gave an APIC IP address of 1.2.3.4 (unlikely to be correct)  Now, if you don't have a credentials.py file, the user will be asked to provide the credentials through interactive prompts.

**Comment by timwukp** [comment]
thanks I had added the file into my project.
